gyp/PRESUBMIT.py

```diff
@@ -1,138 +0,0 @@
-# Copyright (c) 2012 Google Inc. All rights reserved.
```

Comment:
It would be good to have a https://pre-commit.com but we can do that in a separate PR.